### DISCUSSION TITLE
sortable no PoTableColumn não está funcionando

### DISCUSSION THREAD

**Discussion by airak** [po-ui/po-angular#1010] [Tire suas dúvidas] [answered]
Bom dia pessoal, tudo joia?

Ao usar o sortable estou obtendo o seguinte erro:

"Argument of type '{ width: string; property: string; label: any; sortable: boolean; type: string; icons: ({ action: (value: any, cotacao: CotacaoModel) => void; color: string; icon: string; tooltip: any; value: string; } | { action: undefined; color: string; icon: string; tooltip: any; value: string; })[]; }' is not assignable to parameter of type 'PoTableColumn'.
  Object literal may only specify known properties, and 'sortable' does not exist in type 'PoTableColumn'."

Eu estou usando da mesma forma que se encontra no exemplo abaixo da [documentação](https://po-ui.io/documentation/po-table):

```
{
      property: 'favorite',
      label: 'Actions',
      type: 'icon',
      sortable: false,
      icons: [
        {
          action: this.favorite.bind(this),
          color: this.isFavorite.bind(this),
          icon: 'po-icon-star',
          tooltip: 'Favorite',
          value: 'favorite'
        },
        {
          action: this.goToDocumentation.bind(this),
          disabled: this.canGoToDocumentation.bind(this),
          icon: 'po-icon-export',
          tooltip: 'Click to go to documentation',
          value: 'documentation'
        }
    ]
}
```

**Comment by jhonyeduardo** [comment, marked as answer]
Bom dia @Airak 

Qual vesão do PO UI você esta utilizando?

**Reply by airak** [reply]
Estou usando a 5.2.1

**Reply by jhonyeduardo** [reply]
Foi liberado na versão [5.13.0](https://github.com/po-ui/po-angular/blob/master/CHANGELOG.md#5130-2021-09-13)
